pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect } from 'react';
+import { Button, Flex, Modal, Popover } from "@patternfly/react-core";
+import { ExternalLinkSquareAltIcon, HelpIcon, InProgressIcon } from '@patternfly/react-icons';
+
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+import { ShutdownModal } from 'cockpit-components-shutdown.jsx';
+import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { TunedDialogBody } from '../../../pkg/tuned/change-profile.jsx';
+
+import "./cryptoPolicies.scss";
+
+const _ = cockpit.gettext;
+
+// Found in /usr/share/crypto-policies/policies/
+const cryptopolicies = {
+    DEFAULT: _("Recommended, secure settings for current threat models."),
+    FUTURE: _("Protects from anticipated near-term future attacks at the expense of interoperability."),
+    LEGACY: _("Higher interoperability at the cost of an increased attack surface."),
+};
+const applyNeedRebootStamp = "/run/cockpit/crypto-policies-reboot-stamp";
+const displayProfileText = profile => profile.charAt(0) + profile.slice(1, profile.length).toLowerCase();
+
+export const CryptoPolicyRow = () => {
+    const [currentCryptoPolicy, setCurrentCryptoPolicy] = useState(null);
+    const [isOpen, setIsOpen] = useState(false);
+    const [permission, setPermission] = useState(null);
+
+    useEffect(() => {
+        // Avoid cockpit.spawn every re-render from parent
+        if (currentCryptoPolicy === null) {
+            cockpit.spawn(["update-crypto-policies", "--show"], { err: "message" }).then(output => {
```

Comment:
Please move the `.then` to next line, so that it aligns with the `.catch`. (Odd, normally ESLint complains about multiple chained function calls on one line..).

Also, not necessary to introduce a { } block.